src/app/layout.tsx

```diff
@@ -29,29 +28,8 @@ export default function RootLayout({
 			<body className={spaceGrotesk.className}>
 				<Web3ModalProvider initialState={initialState}>
 					<div className='min-h-screen flex flex-col gap-24 relative'>
-						<Image
-							className='absolute top-28 -z-10'
-							src='/images/arcs/1.svg'
-							width={116}
-							height={369}
-							alt='arc-1'
-						/>
-						<Image
-							className='absolute top-0 left-2/3 -z-10'
-							src='/images/arcs/2.svg'
-							width={256}
-							height={109}
-							alt='arc-2'
-						/>
-						<Image
-							className='absolute top-96 right-0 -z-10'
-							src='/images/arcs/3.svg'
-							width={116}
-							height={369}
-							alt='arc-3'
-						/>
```

Comment:
please keep them in the layout, because we have them on all views.

Reply:
Project details view don't have them

![image](https://github.com/Giveth/DeVouch-FE/assets/18689480/3f850059-0cfa-48fe-8a2d-6f01684c6835)